helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -148,28 +148,10 @@ controller:
   resources:
 
     # controller.resources.requests
-    # These are default resource requests when the cluster profile is not
-    # known or when the cluster is larger than "small"
+    # These are default resource requests.
     requests:
-      cpu: 2
-      memory: 2.5Gi
-
-  # controller.profile
-  # Contains resource request profiles that could get selected based on the
-  # value of cluster.profile.
-  # A profile is a set of preconfigured values for the
-  # "spec.template.spec.containers.resources.requests" field of the Deployment
-  # resource.
-  profile:
-    # xxs (1) cluster profile - 1 worker node only - resource requests are not being set
-    # xs (2) cluster profile - 2-3 worker nodes, max CPU < 4 - resource requests are not being set
-    # TODO set resource requests same as small once coredns scale and resources are cluster profile adjusted https://github.com/giantswarm/giantswarm/issues/9316
-    # small (3) cluster profile - > 3 worker nodes, max CPU still < 4
-    small:
-      resources:
-        requests:
-          cpu: 500m
-          memory: 600Mi
+      cpu: 100m
```

Comment:
I'm not sure if those a really good values for our use cases. Maybe we should reduce requests to the former defaults for `profile.small` in a first step.

Reply:
Upstream seems to link to a issue thread here: https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml#L351=

I haven't read through it but if you can find a reason why these are too small, I'm not against increasing limits

Reply:
I read the linked issue. Basically someone is running nginx without requests and wondering why another application consumes all the CPU on the node which slows down nginx health check and makes kubelet kill the nginx pod since it's not responding to health checks in an appropriate time. So I guess those requests are mainly set for health check and giving Kubernetes a possibility to safely schedule nginx.

Reply:
I'd therefore reduce our default requests to the former `small` profile since I guess some of our customers are heavily using nginx and don't want Kubernetes to schedule it on a already heavy loaded node just because nginx requested only 100m of CPU.